.travis.yml

```diff
@@ -1,2 +1,6 @@
 language: go
+# Skip the default install step to avoid running godep: https://docs.travis-ci.com/user/languages/go/#godep-support
```

Comment:
Strange, the docs say that Travis won't run "go get" if `Makefile` is present... Do we know why your change is still needed?

Reply:
It's `godep` that's failing, not `go get`. `godep` runs regardless of Makefile and before `go get`.